TypeDBConsole.java

```diff
@@ -579,24 +582,26 @@ private void runClose(TypeDBTransaction tx) {
         else printer.info("Transaction closed");
     }
 
-    private boolean runSource(TypeDBTransaction tx, String file) {
+    private boolean runSource(TypeDBTransaction tx, String file, boolean printAnswers) {
         try {
             String queryString = new String(Files.readAllBytes(Paths.get(file)), StandardCharsets.UTF_8);
-            return runQuery(tx, queryString);
+            return runQuery(tx, queryString, printAnswers);
         } catch (IOException e) {
             printer.error("Failed to open file '" + file + "'");
             return false;
         }
     }
 
-    private boolean runQuery(TypeDBTransaction tx, String queryString) {
+    @SuppressWarnings("CheckReturnValue")
+    private boolean runQuery(TypeDBTransaction tx, String queryString, boolean printAnswers) {
```

Comment:
We're making the method to behave quite differently depending on the value of `printAnswers`:

- If `false`, then execute the methods asynchronously
- If `true`, then execute the queries synchronously. When this is the case the `running` list is effectively ignored.

Is there a better way of doing it?

Reply:
Comments addressed by @flyingsilverfin in https://github.com/vaticle/typedb-console/pull/175/commits/a8641bbf8e6e15e2b139d7882e6d0c53e5469c0c, https://github.com/vaticle/typedb-console/pull/175/commits/8603c628955034cb34eb402ce19c30c75d937fa0, https://github.com/vaticle/typedb-console/pull/175/commits/f8cdb292fdb846b9ef5b68783382f66c0fe4c239.